### ISSUE TITLE
Error when upgrading to version 1.0.0

### ISSUE THREAD

**Issue by ravenfeld** [alexzhirkevich/qrose#5]
Hello, I was on the `1.0.0-beta3` version and I wanted to upgrade to the `1.0.0` version. But I got the following error.

It may have come from me, if I put back the `1.0.0-beta3` version. No problem
```
Duplicate class io.github.alexzhirkevich.qrose.CachedPainter found in modules qrose-core-release-runtime (io.github.alexzhirkevich:qrose-core-android:1.0.0) and qrose-release-runtime (io.github.alexzhirkevich:qrose-android:1.0.0)
Duplicate class io.github.alexzhirkevich.qrose.CachedPainter$block$1 found in modules qrose-core-release-runtime (io.github.alexzhirkevich:qrose-core-android:1.0.0) and qrose-release-runtime (io.github.alexzhirkevich:qrose-android:1.0.0)
Duplicate class io.github.alexzhirkevich.qrose.ConvertersKt found in modules qrose-core-release-runtime (io.github.alexzhirkevich:qrose-core-android:1.0.0) and qrose-release-runtime (io.github.alexzhirkevich:qrose-android:1.0.0)
Duplicate class io.github.alexzhirkevich.qrose.DrawCache found in modules qrose-core-release-runtime (io.github.alexzhirkevich:qrose-core-android:1.0.0) and qrose-release-runtime (io.github.alexzhirkevich:qrose-android:1.0.0)
Duplicate class io.github.alexzhirkevich.qrose.ToByteArrayKt found in modules qrose-core-release-runtime (io.github.alexzhirkevich:qrose-core-android:1.0.0) and qrose-release-runtime (io.github.alexzhirkevich:qrose-android:1.0.0)
```

**Comment by alexzhirkevich** [comment]
should be fixed in 1.0.1

**Comment by idrisalshikh** [comment]
I encountered this error because I used the version specified in the README file, which is still 1.0.0. It would be helpful if you could update the README file to prevent others from facing the same issue.
